gallery/package.json

```diff
@@ -69,7 +70,8 @@
     "ramda": "0.26.1",
     "react": "16.10.1",
     "react-dom": "16.10.1",
-    "react-router-dom": "=5.0.0",
+    "react-router": "^5.1.0",
```

Comment:
`react-router` should have been an explicit dependency before, but it wasn't and we were relying on it being a transitive dependency of `react-router-dom`.  Additionally, this PR upgrades the react-router packages because the `useLocation` hook that we use was introduced in 5.1.0